test/functional/cursor.test.js

```diff
@@ -405,74 +406,66 @@ describe('Cursor', function () {
           }
 
           function f() {
-            var number_of_functions = 9;
-            var finished = function () {
+            var number_of_functions = 7;
+            var finished = function (cursor) {
               number_of_functions = number_of_functions - 1;
               if (number_of_functions === 0) {
-                done();
+                cursor.close(done);
+              } else {
+                cursor.close();
               }
             };
 
             var cursor = collection.find().sort(['a', 1]);
-            test.deepEqual(['a', 1], cursor.sortValue);
-            finished();
+            test.deepEqual({ a: 1 }, cursor.sortValue);
```

Comment:
these had to change, it no longer just places the value it formats & validates it.